docs-web/configuration/handling-host-collisions.md

```diff
@@ -0,0 +1,67 @@
+# Handling Host Collisions
+
+A host collision occurs when multiple resources configure the same `host`. The Ingress Controller supports two options for handling host collisions:
+* Choosing the winner so that only one resource handles the host.
+* Merging configuration of the conflicting resources.
+
+## Choosing the Winner
+
+Consider the following two resources:
+* `cafe-ingress` Ingress:
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+      name: cafe-ingress
+      annotations:
+        kubernetes.io/ingress.class: "nginx"
+    spec:
+      rules:
+      - host: cafe.example.com
+        . . .
+    ```
+* `cafe-virtual-server` VirtualServer:
+    ```yaml
+    apiVersion: k8s.nginx.org/v1
+    kind: VirtualServer
+    metadata:
+      name: cafe-virtual-server
+    spec:
+      host: cafe.example.co
+      . . .
+    ```
+
+If a user creates both resources in the cluster, a host collision will occur. As a result, the Ingress Controller will pick the winner using the following algorithm:
+
+> If multiple resources contend for the same host, the Ingress Controller will pick the winner based on the `creationTimestamp` of the resources: the oldest resource will win. In case there are more than one oldest resources (their `creationTimestamp` is the same),  the Ingress Controller will choose the resource with the smallest `uid`. Note: the `creationTimestamp` and `uid` fields are part of the resource [ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#objectmeta-v1-meta). 
```

Comment:
```suggestion
> If multiple resources contend for the same host, the Ingress Controller will pick the winner based on the `creationTimestamp` of the resources: the oldest resource will win. If the `creationTimestamp` of both resources is the same, the Ingress Controller will choose the resource with the lower `uid`. 

> Note: The `creationTimestamp` and `uid` fields are part of the resource's [ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#objectmeta-v1-meta). 
```

Reply:
> If the `creationTimestamp` of both resources...

in theory, more than 2 resources can have the same `creationTimestamp`, so that's why I didn't use `both` there.

> choose the resource with the lower `uid`

here are examples of `uid` - `dd50953e-3910-4d0a-8971-17a75b1aa1d5`, `bd6b403b-7f26-482c-8c06-23f11de24a73`.

 I don't think `lower` would apply here, because uids are not numbers like `5` and `6`.

I see how my original description `smaller` could be confusing as well. How about `lexicographically smaller`?  This reflects that to determine the winner in that case, we need to compare strings lexicographically.